hoomd/mpcd/RadialSolventVelocityAnalyzer.cc

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) 2009-2019 The Regents of the University of Michigan
+// This file is part of the HOOMD-blue project, released under the BSD 3-Clause License.
+
+// Maintainer: mphoward
+
+/*!
+ * \file mpcd/RadialSolventVelocityAnalyzer.cc
+ * \brief Declaration of mpcd::RadialSolventVelocityAnalyzer
+ */
+
+#include "RadialSolventVelocityAnalyzer.h"
+#include "hoomd/extern/pybind/include/pybind11/stl.h"
+
+namespace mpcd
+{
+
+/*!
+ * \param sysdata MPCD system data
+ * \param bin_width Width for binning particle distances
+ */
+RadialSolventVelocityAnalyzer::RadialSolventVelocityAnalyzer(std::shared_ptr<mpcd::SystemData> sysdata,
+                                                             Scalar bin_width)
+    : m_mpcd_sys(sysdata),
+      Analyzer(m_mpcd_sys->getSystemDefinition()),
```

Comment:
Call the base constructor first
```suggestion
    : Analyzer(sysdata->getSystemDefinition()),
      m_mpcd_sys(sysdata),
```